crates/sui-graphql-rpc/src/test_infra/cluster.rs

```diff
@@ -203,4 +203,8 @@ impl ExecutorCluster {
         .await
         .expect("Timeout waiting for indexer to catchup to checkpoint");
     }
+
+    pub async fn force_object_snapshot_catchup(&self) {
```

Comment:
Could you add some documentation here explaining that this should only be called in test situations? (Maybe change the name as well to make that clear at the call-site?)